sites/svelte-5-preview/src/routes/docs/content/03-appendix/02-breaking-changes.md

```diff
@@ -4,6 +4,28 @@ title: Breaking changes
 
 While Svelte 5 is a complete rewrite, we have done our best to ensure that most codebases can upgrade with a minimum of hassle. That said, there are a few small breaking changes which may require action on your part. They are listed here.
 
+
+## Codemods
+
+To assist with the upgrade from Svelte 4 to Svelte 5, [Codemod](https://github.com/codemod-com/codemod) provide automated transformations for updating your code to align with many of the new APIs and patterns introduced in Svelte 5.
+
+These following codemods are available : 
+
+- `svelte/5/components-as-functions`
```

Comment:
It would be nice to link to the pages for these
```suggestion
- [`svelte/5/components-as-functions`](https://codemod.com/registry/svelte-5-components-as-functions)
```